cmd/skupper/skupper_link.go

```diff
@@ -140,7 +156,17 @@ func NewCmdLinkStatus(newClient cobraFunc) *cobra.Command {
 						fmt.Println(err)
 						break
 					} else if link.Connected {
-						fmt.Printf("Link %s is active", link.Name)
+						if verbose {
```

Comment:
If a link is not active, the verbose has no effect.
Is this a desired outcome?

Reply:
You are right, I'll fix it.